### PR TITLE
Revise admin ownerships edit

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
Image 6-16-19 at 8 48 PM

**Comment by Robertgervais** [comment]
Image 6-16-19 at 8 54 PM

File: app/controllers/admin/ownerships_controller.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ class Admin::OwnershipsController < Admin::BaseController
   before_filter :find_ownership
 
   def edit
+    render layout: "new_admin"
```
Comment (sethherr) [inline app/controllers/admin/ownerships_controller.rb]: Is this controller wide?

Comment (Robertgervais) [reply app/controllers/admin/ownerships_controller.rb]: No but it should be, i'll fix.

**Comment by Robertgervais** [comment]
#175

**Pull request merged**